doc/api/next_api_changes/deprecations.rst

```diff
@@ -47,3 +47,9 @@ PDF and PS character tracking internals
 The ``used_characters`` attribute and ``track_characters`` and
 ``merge_used_characters`` methods of `.RendererPdf`, `.PdfFile`, and
 `.RendererPS` are deprecated.
+
+Passing raw data to ``register_cmap()``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Passing raw data via parameters *data* and *lut* to ``register_cmap()`` is
+deprecated. Instead, create the colormap explicitly using
+``register_cmap(cmap=LinearSementedColormap(name, data, lut))``.
```

Comment:
Might be nice to have intershpinx links to `register_cmap` and `LinearSegmentedColormap` here?